scripts/encapp.py

```diff
@@ -656,9 +661,31 @@ def main(argv):
         input_filepath = options.replace['input']['filepath']
         if (input_filepath != 'camera' and
                 not encapp_tool.ffutils.video_is_raw(input_filepath)):
```

Comment:
This actually does the opposite of what the label says. This line will _prevent_ the usage of an encoded file.
There will need to be a setting which says to decode the video explicitly (or by parsing the test definition to see if it says to not use surfaces explicitly).

Reply:
Note sure I get this one: This checks that the input filepath is neither "camera" nor a raw file. In that case, it's an encoded file. The following code decodes it into a raw file for the encoder.